modules/quickstart-sap/pages/prepare-clients.adoc

```diff
@@ -47,15 +47,21 @@ Do not use a client machine to host the shared storage disk.
 
 == Download the SAP Installation Software
 
-Download the {sles}{nbsp}15 with SAP installation media from {scc}, and save a copy on each clients.
-Ensure you have saved it in the same file path on each client.
+Download the SAP installation media and save a copy on each clients.
+The software that you require will differ, depending on your environment.
+For example, if you are using HANA, you need the SAP HANA platform.
+If you are using Netweaver, you need several different packages.
+These software packages are provided by SAP, not by {suse}.
+
+Ensure you have saved the installation software in the same file path on each client.
```

Comment:
"file path" is unusual wording (at least, in the context of GNU).  We usually have files and directories.  Path should be used for search path and similar things.  In the shell we have a path such as:
```
PATH=/home/ke/bin:/usr/local/bin:/usr/bin:/bin
```
That's a list of directories.

Reply:
Good point. Updated.